src/Graphics/Texture.cpp

```diff
@@ -306,4 +306,36 @@ bool Texture::blitWithAlpha(Texture* blitMask, int maskOffsetX, int maskOffsetY)
     return true;
 }
 
+// static
+std::unique_ptr<Texture> Texture::generateTextureForNumber(
+    unsigned int number,
+    unsigned int maxLength,
+    Texture* symbolSource,
+    unsigned int charWidth,
+    unsigned int charHeight,
+    unsigned int xOffsetByColor,
+    bool isSigned)
+{
+    // number as text
+    const auto charsCount = maxLength + (isSigned ? 1 : 0);
+    auto texture = std::unique_ptr<Texture>(new Texture(charWidth * charsCount, charHeight));
```

Comment:
I mean that we can define our own std::make_unique function which can be replaced in future when C++14 is ready